Dockerfile

```diff
@@ -0,0 +1,16 @@
+# Use a base image that supports Python 3.11
+FROM python:3.11-slim
+
+WORKDIR /model-service
+
+# Copy the files into the container
+COPY poetry.lock pyproject.toml /model_service/app.py /model-service/
+
+# Install Poetry and dependencies
+RUN pip install poetry \
+    && poetry config virtualenvs.create false \
```

Comment:
Why is this needed?

Reply:
This was mainly for if you run the containe standalone to ensure the dependencies are there. Im not sure if its needed once composed.